docs/contribute/run-vale.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Run the spelling and style linter
+sidebar_position: 7
+---
+
+# Run the spelling and style linter
+
+The documentation suite uses [Vale](https://vale.sh/) as the style guide and spelling linter.
+Vale is currently integrated into the continuous integration (CI) pipeline and is executed on each
+pull request (PR) using a GitHub action. You can select the **Details** link to view the failures.
+
+:::info important
+The Vale GitHub workflow runs on all Markdown files in the document, not only the one's you've updated.
+Failures in the workflow won't prevent your PR from being merged.
+
+You can run Vale locally to view issues directly related to your PR.
+:::
+
+<p align = "center">
+![Vale CI](../assets/vale-ci.png)
+</p>
+
+## Run locally
+
+Run Vale locally to view issues related to the Markdown files you're working on. You can run Vale using the
+command line, or you can integrate it into a [supported editor](https://vale.sh/docs/integrations/guide/) to
+view issues in real-time.
+
+### Use the command line
+
+1. [Install Vale locally](https://vale.sh/docs/vale-cli/installation/#package-managers).
+
+1. Checkout the repo containing our Vale settings:
+
+   ```bash
+   git clone https://github.com/Consensys/docs-gha.git
+   ```
+
+1. Override the default location of the `.vale.ini` file by setting the `VALE_CONFIG_PATH` environment
+    variable to the location of the file in the repo. For example, on macOS this is:
+
+    ```bash
+    export VALE_CONFIG_PATH="/Users/jdoe/documentation/docs-gha/spelling/.vale.ini"
```

Comment:
```suggestion
    export VALE_CONFIG_PATH="/Users/{user-name}/documentation/docs-gha/spelling/.vale.ini"
```

Reply:
alt is username like cherry-blossom
Avoid jdoe --> John Doe is the boilerplate name for an unidentified male corpse(!)